appengine_config.py

```diff
@@ -74,6 +74,13 @@
 requests_toolbelt.adapters.appengine.monkeypatch()
 old_get_distribution = pkg_resources.get_distribution # pylint: disable=invalid-name
 
+# Disables "AppEnginePlatformWarning" because it is a "warning" that occurs
```

Comment:
Please add a link to a relevant SO post or similar that explains why it is OK to do this, and when we can remove it (IIRC it's after we migrate to Python 3? If so, maybe file a TODO/issue for that).

Reply:
Hi @seanlip, from what I can tell, it looks like this error will not go away unless we use Sockets (which is a billable service) and I'm not sure if we plan to use Sockets in the future. So I have linked an SO answer says why this is safe to ignore but have not added a TODO to get rid of this.

References:
https://issuetracker.google.com/issues/35907699#comment9
https://urllib3.readthedocs.io/en/latest/reference/contrib/appengine.html#module-urllib3.contrib.appengine